src/tool_cb_wrt.c

```diff
@@ -48,17 +53,14 @@ bool tool_create_output_file(struct OutStruct *outs,
 
   if(outs->is_cd_filename) {
     /* don't overwrite existing files */
-    file = fopen(outs->filename, "rb");
-    if(file) {
-      fclose(file);
-      warnf(global, "Refusing to overwrite %s: %s\n", outs->filename,
-            strerror(EEXIST));
-      return FALSE;
-    }
+    int fd = open(outs->filename, O_CREAT | O_WRONLY | O_EXCL, 0666);
```

Comment:
I think since fdopen has to open the same way as open you would need O_BINARY | O_WRONLY

Reply:
Also I'm not sure if 0666 is entirely compatible perhaps you should use [macros](https://pubs.opengroup.org/onlinepubs/007904975/basedefs/sys/stat.h.html) instead, though on Windows I have S_IREAD | S_IWRITE but not S_IRUSR | S_IWUSR